src/controllers/api.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response } from 'express';
+import { mongoose } from '../config/database';
+import { users } from '../models/users';
+
+const db = mongoose.connection;
+
+export let employees = (req: Request, res: Response) => {
+	const query = users.find({}, (err: Error, data: [object]) => {
```

Comment:
Instead of "data: [object]" is this supposed to be "data: object[]". I know it passes but this is not typescript syntax unless it is some rule I don't know.

Reply:
I thought you guys worked on it? did you not push?